### PR TITLE
cloud: refine performance reference

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by joey-yez — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 600     | 32,266 | 29,168 | 548          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS    | QPS    | Latency (ms) |
+    |-------------------|---------|--------|--------|---------------|
+    | Insert            | 600     | 17,831 | 17,831 | 34            |
+    | Point Select      | 600     | 93,287 | 93,287 | 6             |
+    | Read Write        | 300     | 29,729 | 1,486  | 202           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:149]:
```suggestion
    | Read Write        | 300     | 1,486  | 29,729 | 202           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 33,394 | 30,188 | 1,048        |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS    | QPS    | Latency (ms) |
+    |-------------------|---------|--------|--------|---------------|
+    | Insert            | 2,000   | 23,633 | 23,633 | 84            |
+    | Point Select      | 600     | 93,287 | 93,287 | 6             |
+    | Read Write        | 600     | 30,464 | 1,523  | 394           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:167]:
```suggestion
    | Read Write        | 600     | 1,523  | 30,464 | 394           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 62,918 | 56,878 | 310          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 1,200   | 33,892  | 33,892  | 23            |
+    | Point Select      | 1,200   | 185,574 | 181,255 | 4             |
+    | Read Write        | 600     | 59,160  | 2,958   | 127           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:187]:
```suggestion
    | Read Write        | 600     | 2,958   | 59,160  | 127           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 2,400   | 65,452 | 59,169 | 570          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 4,000   | 47,029  | 47,029  | 43            |
+    | Point Select      | 1,200   | 185,574 | 181,255 | 4             |
+    | Read Write        | 1,200   | 60,624  | 3,030   | 197           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:205]:
```suggestion
    | Read Write        | 1,200   | 3,030   | 60,624  | 197           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 67,941 | 61,419 | 540          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 1,200   | 35,096  | 35,096  | 34            |
+    | Point Select      | 1,200   | 228,600 | 228,600 | 5             |
+    | Read Write        | 600     | 73,150  | 3,658   | 164           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:242]:
```suggestion
    | Read Write        | 600     | 3,658   | 73,150  | 164           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 67,941 | 61,419 | 540          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 2,000   | 43,338  | 43,338  | 46            |
+    | Point Select      | 1,200   | 228,600 | 228,600 | 5             |
+    | Read Write        | 1,200   | 73,631  | 3,682   | 326           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:260]:
```suggestion
    | Read Write        | 1,200   | 3,682   | 73,631  | 326           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|---------|---------|--------------|
+    | TPCC              | 2,400   | 133,164 | 120,380 | 305          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 2,400   | 69,139  | 69,139  | 22            |
+    | Point Select      | 2,400   | 448,056 | 448,056 | 4             |
+    | Read Write        | 1,200   | 145,568 | 7,310   | 97            |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:280]:
```suggestion
    | Read Write        | 1,200   | 7,310   | 145,568 | 97            |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+    |-------------------|---------|---------|---------|--------------|
+    | TPCC              | 2,400   | 133,164 | 120,380 | 305          |
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS     | QPS     | Latency (ms) |
+    |-------------------|---------|---------|---------|---------------|
+    | Insert            | 4,000   | 86,242  | 86,242  | 25            |
+    | Point Select      | 2,400   | 448,056 | 448,056 | 4             |
+    | Read Write        | 2,400   | 146,526 | 7,326   | 172           |
```
Comment (joey-yez) [inline tidb-cloud/tidb-cloud-performance-reference.md:298]:
```suggestion
    | Read Write        | 2,400   | 7,326   | 146,526 | 172           |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+
+    Sysbench OLTP performance:
+
+    | Transaction model | Threads | TPS    | Latency (ms) |
+    |-------------------|---------|--------|--------|
+    | Insert            | 100     | 6147  | 14.77        |
+    | Point Select      | 100     | 19462 | 3.21         |
+    | Read Write        | 100     | 378 | 9.58         |
+    | Update Index      | 100     | 3140 | 30.34        |
+    | Update Non-index  | 100     | 5805  | 15.92 |
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-performance-reference.md:62]:
```suggestion
    | Insert            | 100     | 6,147  | 14.77        |
    | Point Select      | 100     | 19,462 | 3.21         |
    | Read Write        | 100     | 378 | 9.58         |
    | Update Index      | 100     | 3,140 | 30.34        |
    | Update Non-index  | 100     | 5,805  | 15.92 |
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+
+Currently, the 2 vCPU support of TiDB and TiKV is still in beta.
+
+Test environment:
+
+- TiDB version: v6.1.0
+- Warehouses: 1000
+- Data size: 80 G
+- Table size: 10000000
+- Table count: 32
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-performance-reference.md:20]:
```suggestion
- TiDB version: v6.1.0
- Warehouses: 1,000
- Data size: 80 GiB
- Table size: 10,000,000
- Table count: 32
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 11]
```diff
@@ -0,0 +1,300 @@
...
+
+## 4 vCPU performance
+
+Test environment:
+
+- TiDB version: v5.4.0
+- Warehouses: 5000
+- Data size: 366 G
+- Table size: 10000000
+- Table count: 16
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-performance-reference.md:72]:
```suggestion
- TiDB version: v5.4.0
- Warehouses: 5,000
- Data size: 366 GiB
- Table size: 10,000,000
- Table count: 16
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 12]
```diff
@@ -0,0 +1,300 @@
...
+
+## 8 vCPU performance
+
+Test environment:
+
+- TiDB version: v5.4.0
+- Warehouses: 5000
+- Data size: 366 G
+- Table size: 10000000
+- Table count: 16
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-performance-reference.md:124]:
```suggestion
- TiDB version: v5.4.0
- Warehouses: 5,000
- Data size: 366 GiB
- Table size: 10,000,000
- Table count: 16
```

File: tidb-cloud/tidb-cloud-performance-reference.md [hunk 13]
```diff
@@ -0,0 +1,300 @@
...
+
+## 16 vCPU performance
+
+Test environment:
+
+- TiDB version: v5.4.0
+- Warehouses: 5000
+- Data size: 366 G
+- Table size: 10000000
+- Table count: 16
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-performance-reference.md:217]:
```suggestion
- TiDB version: v5.4.0
- Warehouses: 5,000
- Data size: 366 GiB
- Table size: 10,000,000
- Table count: 16
```

File: TOC-tidb-cloud.md [hunk 14]
```diff
@@ -75,6 +75,7 @@
   - Plan Your Cluster
     - [Select Your Cluster Tier](/tidb-cloud/select-cluster-tier.md)
     - [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md)
+    - [TiDB Cloud Performance Reference](/tidb-cloud/tidb-cloud-performance-reference.md)
```
Comment (Oreoxmt) [inline TOC-tidb-cloud.md:78]:
```suggestion
    - [TiDB Cloud Performance Test Report](/tidb-cloud/tidb-cloud-performance-reference.md)
```
or
```suggestion
    - [TiDB Cloud Performance Benchmarks](/tidb-cloud/tidb-cloud-performance-reference.md)
```
or
```suggestion
    - [TiDB Cloud Benchmarks](/tidb-cloud/tidb-cloud-performance-reference.md)
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: TOC-tidb-cloud.md (hunk 14 above)
Comment (qiancai) [reply TOC-tidb-cloud.md:78]: After checking with QA, the current doc does not contain any test steps, so it cannot be defined as a test result or benchmark docs.

**Pull request merged**